src/components/Results/StudyHeader.tsx

```diff
@@ -20,27 +20,21 @@ import TargetIcon from './TargetIcon';
 import UnsaveIcon from './UnsaveIcon';
 import BookmarkCheckIcon from './BookmarkCheckIcon';
 
-import { StudyDetailProps, SaveStudyHandler, ContactProps } from './types';
+import { StudyDetailProps, SaveStudyHandler } from './types';
 import StudyDetailsButton from './StudyDetailsButton';
 
 type StudyHeaderProps = {
   isExpanded: boolean;
   study: StudyDetailProps;
   handleSaveStudy: SaveStudyHandler;
   isStudySaved: boolean;
-  closestFacility: ContactProps;
 };
 
-const StudyHeader = ({
-  isExpanded,
-  study,
-  handleSaveStudy,
-  isStudySaved,
-  closestFacility,
-}: StudyHeaderProps): ReactElement => {
+const StudyHeader = ({ isExpanded, study, handleSaveStudy, isStudySaved }: StudyHeaderProps): ReactElement => {
   const studyTags = [...study.conditions, study.phase, study.type];
   const theme = useTheme();
   const isExtraSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+  const closestFacility = study?.closestFacilities?.[0];
```

Comment:
I would just go ahead and call this `closestFacilityDistance` (or something of the sort) and set this to `study?.closestFacilities?.[0]?.distance` and use it as is on lines 111 and 114 since you're not using the other fields of the closestFacility from what I can see.